### DISCUSSION TITLE
How do I set up proxy server on hotspot?

### DISCUSSION THREAD

**Discussion by gtxg16** [termux/termux-app#2473] [Q&A]
Ok, so I want to connect termux to my mobile hotspot, and set up a proxy server. So when another device connects to my hotspot it can use the proxy server currently running in termux as a proxy. I also need to set up local DNS and static IP in termux. I only know how to do the latter. The former, I will just research after I find this one out

**Comment by SDRausty** [comment]
>How do I set up proxy server on hotspot? 

Hopefully this link will help https://www.debian.org/doc/manuals/debian-reference/ch05.en.html
